cypress/integration/userCanVoteOnAJoke.feature.js

```diff
@@ -0,0 +1,59 @@
+describe("Authenticated user can vote on a Joke", () => {
+  beforeEach(() => {
+    cy.server();
+    cy.route({
+      method: "POST",
+      url: "http://localhost:3000/api/v1/auth/sign_in",
+      response: "fixture:login.json",
+      headers: {
+        uid: "user@mail.com",
+      },
+    });
+    cy.route({
+      method: "GET",
+      url: "http://localhost:3000/api/v1/joke",
+      response: "fixture:randomJokeResponse.json",
+    });
+    cy.route({
+      method: "POST",
+      url: "http://localhost:3000/api/v1/votes",
+      response: "fixture:voteJokeResponse.json",
+      headers: {
+        uid: "user@mail.com",
+      },
+    });
+
+    cy.visit("/");
+
+    cy.get('[data-cy="login"]').click();
+    cy.get('[data-cy="login-form"]').within(() => {
+      cy.get('[data-cy="email"]').type("user@mail.com");
+      cy.get('[data-cy="password"]').type("password");
+      cy.get('[data-cy="button"]').contains("Submit").click();
+    });
+    cy.get('[data-cy="message"]').should("contain", "Hi user@mail.com");
```

Comment:
you are already testing this in the other test, please remove this

Reply:
No, they shouldn't be able to? Now they can't right?